docs/DeveloperGuide.adoc

```diff
@@ -475,23 +475,34 @@ is rather high. Implementing alternative 2 would be more tedious on the user's s
 // tag::list[]
 === List (Mainlist, Blacklist etc) mechanism
 
-List feature is a useful feature to view persons of a certain category. In the context of this project, master list contains all the contacts of the debt collection agency.
+Multiple List feature is a useful feature to view debtors of different categories. Currently these different lists include masterlist and blacklist.
 
-We are displaying multiple lists to suit the need for the direction of the project. The lists could be viewed with the respective commands that will bring it to picture. The commands to materialize these lists are named after the lists itself. Meaning, to view the blacklist, the command typed is "blacklist" or "bl'.
+These lists could be viewed with the respective commands that will update the current displayed panel. The commands to materialize these lists are named after the lists themselves. Meaning, to view the `blacklist`, the command typed is "blacklist" or "bl'.
```

Comment:
- update the panel that is currently displayed.*
- The commands to display these lists *

Reply:
Noted.